### DISCUSSION TITLE
Source Code for .Net Core Showcase

### DISCUSSION THREAD

**Discussion by jasonweier78** [fluentribbon/Fluent.Ribbon#968] [Q&A] [answered]
Hi,

Thank you so much for continuing to support this library, it'll save me a ton of time! 

I was going through the examples and sample code but I don't see the source for the .Net Core 3 version of the Showcase app. Can you point me in the right direction or make it available?

**Comment by batzen** [comment, marked as answer]
There is no separate source code for the different framework version.
The showcase application, as does Fluent.Ribbon itself, uses multitargeting to compile for different frameworks while only using a single csproj file.

**Reply by jasonweier78** [reply]
Ah, of course I see it now. Thanks for the quick response!
